src/background/RemotePontoon.ts

```diff
@@ -116,21 +116,27 @@ async function updateNotificationsIfThereAreNew(pageContent: string) {
   }
 }
 
-async function updateNotificationsData() {
+export async function updateNotificationsData() {
   try {
+    // setting the state to loading
+    await saveToStorage(StorageContent['notificationsDataLoadingState'] = "loading");
```

Comment:
```suggestion
    await saveToStorage({notificationsDataLoadingState: "loading"});
```

Please, if you are not familiar with TypeScript or JavaScript, have a look at https://www.typescriptlang.org/docs/handbook/intro.html and please please go through the development setup I mentioned earlier. Running the build locally will give you error messages for code that is not valid, and other issues I noted in comments as well.